eidoscommon/src/main/scala/org/clulab/wm/eidoscommon/EidosProcessor.scala

```diff
@@ -109,6 +113,32 @@ class EidosPortugueseProcessor(val language: String, cutoff: Int) extends Portug
   def getTagSet: TagSet = tagSet
 }
 
+class EidosCluProcessor(val language: String, cutoff: Int) extends FastNLPProcessorWithSemanticRoles
+  with EidosProcessor with SentencesExtractor with LanguageSpecific {
+  lazy val eidosTokenizer: EidosTokenizer = new EidosTokenizer(localTokenizer, cutoff)
+  override lazy val tokenizer = new EidosTokenizer(localTokenizer, cutoff)
```

Comment:
Is there a merge problem here?  Can tokenizer just be eidosTokenizer from the line above?

Reply:
You are correct! Will fix in next commit.